### DISCUSSION TITLE
is this for amd ryzen 7840hs processor igpu too?

### DISCUSSION THREAD

**Discussion by Greatz08** [ROCm/ROCm#3254] [Q&A] [answered]
new in amd series so asking here :-))

**Comment by darren-amd** [comment, marked as answer]
Hi @Greatz08,

Unfortunately, ROCm is not supported on the Ryzen 7840HS IGPU. You can find the support matrix here: [Support Matrix](https://rocm.docs.amd.com/projects/install-on-linux/en/latest/reference/system-requirements.html).
